webgoat-container/src/main/resources/application-webgoat.properties

```diff
@@ -12,6 +12,7 @@ server.ssl.key-alias=${WEBGOAT_KEY_ALIAS:goat}
 server.ssl.enabled=${WEBGOAT_SSLENABLED:false}
 security.require-ssl=${WEBGOAT_SSLENABLED:false}
 
+hsqldb.port=${WEBGOAT_HSQLPORT:9001}
```

Comment:
You can use ${hsqldb.port} below in spring.datasource.url

Reply:
I just wanted to include this one with the same WEBGOAT_HSQLPORT property to overrule this port. As this is used in other webgoat and webwolf properties. However the WebGoat server ignored this because of hsqldb.port being used in the code but not in the properties. Actually not relevant to the XXE exercise, but still would help in making the test ports dynamic in the future

Reply:
Good idea, can we then change `spring.datasource.url` to:

`jdbc:hsqldb:hsql://${WEBGOAT_HOST:127.0.0.1}:${hsqldb.port}/webgoat` 
?

Reply:
ok